dev/merge_spark_pr.py

```diff
@@ -285,6 +286,56 @@ def resolve_jira_issues(title, merge_branches, comment):
     for jira_id in jira_ids:
         resolve_jira_issue(merge_branches, comment, jira_id)
 
+def standardize_jira_ref(text):
+    # Standardize the [MODULE] SPARK-XXXXX prefix
```

Comment:
Great idea, Josh. Check out the latest diff and see if it's what you had in mind. 
I like what you have in spark-pr-dashboard. Would it make more sense to reference your code from the merge script instead? It's a bit different & would require a bit of finagling, but I hate to have similar code duplicated in different places.